RELEASE-LEADS.md

```diff
@@ -119,104 +109,27 @@ Make a copy of the
-
-Shared dependencies like `knative/{test-infra, pkg, caching, networking}` are
-kept up-to-date nightly in each of the releasing repositories. To stabilize
-things shortly before the release we cut the `release-x.y` branches on those 7
-days prior to the main release.
-
-First, create a release branch for `test-infra` named `release-x.y`.
-
-Next, `pkg` needs to pin to `test-infra`'s release branch. To do that, edit
-`hack/update-deps.sh` in `pkg` **on the newly created branch** to pin the
-branch. Then run `./hack/update-deps.sh --upgrade` and commit the changes.
-
-The change to `hack/update-deps.sh` will look like this:
-
-```diff
-diff --git a/hack/update-deps.sh b/hack/update-deps.sh
-index a39fc858..0634362f 100755
---- a/hack/update-deps.sh
-+++ b/hack/update-deps.sh
-@@ -26,7 +26,7 @@ cd ${ROOT_DIR}
- # The list of dependencies that we track at HEAD and periodically
- # float forward in this repository.
- FLOATING_DEPS=(
--  "knative.dev/test-infra@master"
-+  "knative.dev/test-infra@release-x.y"
- )
+### Cut release branches of supporting repos
 
- # Parse flags to determine any we should pass to dep.
-```
+We need to start cutting release branches in each module that does not produce a
+release artifact, but we will do it from least dependent to most dependent.
+Follow the [cutting release branches](#cutting_release_branches) guide, starting
+with the `hack` repo:
 
-PR the changes to each repository respectively, prepending the PR title with
-`[RELEASE]`.
-
-After `test-infra` and `pkg` are pinned, change `caching` and `networking`'s
-`update-deps.sh` to use `release-x.y` branch of `test-infra` and `pkg`.
-Following that, cut new `release-x.y` branches for `caching` and `networking`.
-
-### Pin `test-infra`, `pkg`, `caching`, `networking` in downstream repositories
-
-Similar to how we pin `pkg` to `test-infra`, all downstream users must be pinned
-to the newly cut `release-x.y` branches on those libraries. The changes to
-`hack/update-deps.sh` look similar to above, but in most cases both dependencies
-will need to be pinned.
-
-```diff
-diff --git a/hack/update-deps.sh b/hack/update-deps.sh
-index b277dd3ff..1989885ce 100755
---- a/hack/update-deps.sh
-+++ b/hack/update-deps.sh
-@@ -32,8 +32,8 @@ VERSION="master"
- # The list of dependencies that we track at HEAD and periodically
- # float forward in this repository.
- FLOATING_DEPS=(
--  "knative.dev/test-infra@${VERSION}"
--  "knative.dev/pkg@${VERSION}"
--  "knative.dev/caching@${VERSION}"
--  "knative.dev/networking@${VERSION}"
-+  "knative.dev/test-infra@release-x.y"
-+  "knative.dev/pkg@release-x.y"
-+  "knative.dev/caching@release-x.y"
-+  "knative.dev/networking@release-x.y"
- )
-```
+- [knative/hack](https://github.com/knative/hack)
 
-The downstream repositories this needs to happen on are:
+After `hack`:
 
-- [knative/client](https://github.com/knative/client)
+- [knative/pkg](https://github.com/knative/pkg)
+- [knative/test-infra](https://github.com/knative/test-infra)
```

Comment:
Do we need to keep cutting this?

Reply:
I think we should for 0.19 to make sure we have a version locked copy of the scripts that prow is running

Reply:
it might not be used

Reply:
The scripts themselves are not used by Prow test image so we don't need to cut for test-infra anymore.

Reply:
We can drop that step next release if it becomes true, I just want to make sure we caught everything...